src/common/messages/MessageType.java

```diff
@@ -41,6 +41,14 @@
 	 * A message indicating that the elevator is leaving the floor
 	 */
 	EVELATOR_LEAVING_FLOOR_MESSAGE,
+	
+	/**
+	 * A message indicating that the elevator is leaving the floor
+	 */
+	SCHEDULER_ELEVATOR_COMMAND,
+	
+	ELEVATOR_TRANSPORT_REQUEST,
```

Comment:
ELEVATOR_TRANSPORT_REQUEST and ELEVATOR_FLOOR_REQUEST can easily be misinterpreted.

Should probably JavaDoc to clarify intentions.